test/sanity/validate-modules/main.py

```diff
@@ -375,14 +375,24 @@ def _check_for_sys_exit(self):
                 msg='sys.exit() call found. Should be exit_json/fail_json'
             )
 
-    def _check_for_gpl3_header(self):
-        if ('GNU General Public License' not in self.text and
-                'version 3' not in self.text):
+    def _check_gpl3_header(self):
+        header = '\n'.join(self.text.split('\n')[:20])
+        if ('GNU General Public License' not in header and
+                'version 3' not in header):
```

Comment:
This conditional allows the test to pass if either `GNU General Public License` or `version 3` is present. Shouldn't we require both?

Reply:
Ah, I see.  Yeah, should be 
``` python
if not ('GNU General Public License' in header and 'version 3' in header):
```

Reply:
Fixed